app/services/action_builder.rb

```diff
@@ -12,7 +12,7 @@ def build_action(extra_attrs = {})
       subscribed_member: subscribed_member
     }.merge(extra_attrs))
 
-    ActionQueue::Pusher.push(action)
+    ActionQueue::Pusher.push(:new_action, action)
```

Comment:
I think it makes for a more expressive and consistent interface if we use the format `push(event_name, object)`. This also provides a good pattern to allow the Pusher functionality to be extended.